gatsbyjs/gatsby
# Update prettier

- Update prettier to latest version
- Run `yarn format` from root of project
- Remove prettier-ignore for nested markdown (see #21743)

Closes #22506
